packages/xrpl/src/models/methods/accountObjects.ts

```diff
@@ -46,8 +46,6 @@ export interface AccountObjectsRequest
   account: string
   /**
    * If included, filter results to include only this type of ledger object.
-   * The valid types are: Check , DepositPreauth, Escrow, Offer, PayChannel,
```

Comment:
I removed this part of the docs because it's hard to keep updated (it's already missing a bunch of object types), and IMO it's worse to have this be outdated than to not have it (since the docs have the types).
https://js.xrpl.org/types/AccountObjectType.html